grist_api/grist_api.py

```diff
@@ -117,6 +117,25 @@ def call(self, url, json_data=None, method=None, prefix=None):
           raise resp.raise_for_status()
       return resp
 
+  def call(self, url, json_data=None, method=None, prefix=None):
+    """
+    Low-level interface that returns the _raw_call result as json
+    """
+    result = self._raw_call(url, json_data, method, prefix)
```

Comment:
Minor note: let's use keyword-argument syntax, i.e. `self._raw_call(url, json_data=json_data, method=method, prefix=prefix)` as `_raw_call`, to avoid hard-to-catch surprises if anyone changes the signature of `_raw_call`.